integration/tests/cook/util.py

```diff
@@ -460,7 +460,6 @@ def docker_working_directory():
 def get_default_cpus():
     return float(os.getenv('COOK_DEFAULT_JOB_CPUS', 0.05))
 
-
```

Comment:
This could be a war between autoformatters, but I believe python style wants to put two newlines between functions. If true, can you revert this change?